src/index.css

```diff
@@ -0,0 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+* {
+    font-family: 'Jura', sans-serif;
+    color: #EAEAEA;
+}
+
+@layer components {
+    .btn-primary {
+      @apply w-full  border-2 border-green rounded-md py-1 bg-green text-black mb-4 hover:cursor-pointer hover:bg-green/85 font-semibold;
+    }
+    .input-area{
+        @apply w-full rounded-md pt-2 pl-4 pb-2 text-black focus:outline-none focus:ring focus:ring-blue focus:border-transparent;
+    }
+    .search-area{
+        @apply w-full rounded-md pt-0.5 pl-4 pb-0.5 border-2 ring-2 ring-blue text-black focus:outline-none focus:ring-2 focus:ring-blue  focus:border-transparent;
```

Comment:
Let's apply a _text-blue_ or _text-white_ class for better readability

`@apply w-full rounded-md pt-0.5 pl-4 pb-0.5 border-2 ring-2 ring-blue text-blue focus:outline-none focus:ring-2 focus:ring-blue  focus:border-transparent;
`

Before:
![image](https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/68535fb5-00fb-496b-be34-37cb54a3323d)

After:
![image](https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/ffed8f1c-f0fe-4218-8399-0334dd8f7ece)